### ISSUE TITLE
Font issue

### ISSUE THREAD

**Issue by Loots-it** [stefanzweifel/sidecar-browsershot#97]
There is a difference between the default fonts that are used in sidecar-browsershot and browsershot.

Small example:

```
        $pdfString = Browsershot::html($html)
            ->margins(18, 18, 18, 18)
            ->format('A4')
            ->pdf();
```

The html:
```
<!DOCTYPE html>
<html>
<body>
  <h1>Invoice</h1>
</body>
</html>

```

The outcome in sidecar-browsershot:
[invoice-aws.pdf](https://github.com/stefanzweifel/sidecar-browsershot/files/12873809/invoice-aws.pdf)

The outcome in browsershot:
[invoice.pdf](https://github.com/stefanzweifel/sidecar-browsershot/files/12873804/invoice.pdf)

In retrospective, it wasn't a great idea of me to not specify a font in the first place. But it seems that font support is not the same in sidecar-browsershot as in browsershot. I could probable use a font-face but I want to use a local one. Maybe it's an idea to add such an option to sidecar-browsershot? If I create a pull request for this, would this be considered?

**Comment by stefanzweifel** [comment]
If you don't declare the font family that should be used in your `$html`, then the default [system font stack](https://systemfontstack.com/) of the underlying operating system will be used.

Haven't checked the source of `spatie/browsershot`, but I'm sure that they don't declare a default font or include fonts by default. That would feel weird.

Weirdly I can't reproduce this error in screeenly, my side project that uses thie package under the hood. [This playground](https://3.screeenly.com/playground?data[url]=null&data[file_type]=pdf&data[window_width]=1440&data[window_height]=800&data[timeout]=10&data[css_media_type]=print&data[full_page]=false&data[pdf_show_background]=false&data[disable_javascript]=false&data[dismiss_dialogs]=false&data[wait_until_network_idle]=false&data[disable_web_security]=false&data[hide_background]=false&data[ignore_https_errors]=false&data[html]=%3Ch1%3EDefault+Font+Family%3C%2Fh1%3E%0A%3Ch1+style%3D%22font-family%3A+Arial%3B%22%3EArial%3C%2Fh1%3E%0A%3Ch1+style%3D%22font-family%3A+avenir%3B%22%3Eavenir%3C%2Fh1%3E%0A%3Ch1+style%3D%22font-family%3A+Ubuntu%3B%22%3EUbuntu%3C%2Fh1%3E%0A&data[paper_format]=A4&data[paper_orientation]=null&data[paper_margins_top]=10&data[paper_margins_right]=10&data[paper_margins_bottom]=10&data[paper_margins_left]=10&data[delay]=null&data[selector]=null&data[clip_x]=null&data[clip_y]=null&data[clip_width]=null&data[clip_height]=null&data[http_headers]=null&data[basic_authentication_username]=null&data[basic_authentication_password]=null) generates PDFs that don't use a serif-font if no `font-family` is declared.
(The project uses the latest version of this package 🤔)

---

I'm hesitant to fix as it feels like a niche problem; but feel free to explore how to fix this bug. 
I can't promise that I will merge a PR though, if it puts a high maintenance burden on to me.

**Comment by Loots-it** [comment]
But the given font-family does not seem to work? [This playground](https://3.screeenly.com/playground?data%5Burl%5D=null&data%5Bfile_type%5D=pdf&data%5Bwindow_width%5D=1440&data%5Bwindow_height%5D=800&data%5Btimeout%5D=10&data%5Bcss_media_type%5D=print&data%5Bfull_page%5D=false&data%5Bpdf_show_background%5D=false&data%5Bdisable_javascript%5D=false&data%5Bdismiss_dialogs%5D=false&data%5Bwait_until_network_idle%5D=false&data%5Bdisable_web_security%5D=false&data%5Bhide_background%5D=false&data%5Bignore_https_errors%5D=false&data%5Bhtml%5D=%3Ch1%3EDefault+Font+Family%3C%2Fh1%3E%0A%3Ch1+style%3D%22font-family%3A+Arial%3B%22%3EArial%3C%2Fh1%3E%0A%3Ch1+style%3D%22font-family%3A+avenir%3B%22%3Eavenir%3C%2Fh1%3E%0A%3Ch1+style%3D%22font-family%3A+Ubuntu%3B%22%3EUbuntu%3C%2Fh1%3E%0A&data%5Bpaper_format%5D=A4&data%5Bpaper_orientation%5D=null&data%5Bpaper_margins_top%5D=10&data%5Bpaper_margins_right%5D=10&data%5Bpaper_margins_bottom%5D=10&data%5Bpaper_margins_left%5D=10&data%5Bdelay%5D=null&data%5Bselector%5D=null&data%5Bclip_x%5D=null&data%5Bclip_y%5D=null&data%5Bclip_width%5D=null&data%5Bclip_height%5D=null&data%5Bhttp_headers%5D=null&data%5Bbasic_authentication_username%5D=null&data%5Bbasic_authentication_password%5D=null&data[url]=null&data[file_type]=pdf&data[window_width]=1440&data[window_height]=800&data[timeout]=10&data[css_media_type]=print&data[paper_format]=A4&data[paper_margins_top]=10&data[paper_margins_right]=10&data[paper_margins_bottom]=10&data[paper_margins_left]=10&data[html]=%3Ch1%3EInvoice%3C%2Fh1%3E%0A%3Ch1+style%3D%22font-family%3A+Arial%3B%22%3E+Invoice+%3C%2Fh1%3E%0A%3Ch1+style%3D%22font-family%3A+avenir%3B%22%3E+Invoice+%3C%2Fh1%3E%0A%3Ch1+style%3D%22font-family%3A+Ubuntu%3B%22%3E+Invoice+%3C%2Fh1%3E%0A%3Ch1+style%3D%22font-family%3A+%27Times+New+Roman%27%3B%22%3EInvoice%3C%2Fh1%3E&data[full_page]=false&data[pdf_show_background]=false&data[disable_javascript]=false&data[dismiss_dialogs]=false&data[wait_until_network_idle]=false&data[disable_web_security]=false&data[hide_background]=false&data[ignore_https_errors]=false&data[paper_orientation]=null&data[delay]=null&data[selector]=null&data[clip_x]=null&data[clip_y]=null&data[clip_width]=null&data[clip_height]=null&data[http_headers]=null&data[basic_authentication_username]=null&data[basic_authentication_password]=null) gives me exactly the same font even if I set the font-family. Or maybe it's dependent on the aws regio, I'm not sure if screeenly uses the closest region? I'm using the frankfurt region for my own project. Maybe there is a bug in that specific layer?

**Comment by stefanzweifel** [comment]
@Loots-it Yeah, something is definitely off here. 
screeenly is also hosted in eu-central-1, but I noticed that an older version of the Lambda function is still around and somehow get's traffic. Might be a config-error on my side.

Have to be honest here: no idea which fonts are available on AWS Lambda by default. Weird that `font-family` in the example doesn't seem to have an impact at all. 🤔

**Comment by Loots-it** [comment]
I did some experiments and it's quite easy to add a new font. It makes sense that we don't know which fonts are available on AWS Lambda. The idea is that it's serverless, so we shouldn't depend on any defaults (in my opinion).

I feel like it would be a big benefit to add some default font families to sidecar-browsershot and provide the possibility for users to add more custom fonts. I don't think the development is that much work so the maintenance burden shouldn't be too high. Do you think that this might be worth it? If so, I will start on a PR.

**Comment by stefanzweifel** [comment]
Great to here @Loots-it. 
Feel free to send in a PR.

**Comment by stefanzweifel** [comment]
Support for custom fonts has been implemented in #101.
A new version has been released: https://github.com/stefanzweifel/sidecar-browsershot/releases/tag/v1.13.0
